### ISSUE TITLE
Accessing the paper

### ISSUE THREAD

**Issue by nicolasch96** [wangzhihua520/OF_BIQA#1]
Hello,

I have a few questions concerning the work:

- Is there a way to access the paper (maybe put it on ARXIV)?
-  How does this work differ from [DipIQ](https://arxiv.org/pdf/1904.06505.pdf)?

Thank you in advance,

Nicolas

**Comment by wzhsysu** [comment]
Thank you for following our work

1. you can access the paper via https://www.sciencedirect.com/science/article/abs/pii/S0031320322007750
2. While DipIQ targets synthetic distortions, our work aims to transfer the knowledge gained from studying synthetic distortions to authentic distortions without relying on human labels. 

Best

Zhihua

**Comment by EvanHLZ** [comment]
Hello,

I wonder where is the file named MNL_DA-00008.pt appeared in the code. Because it seems you didn't mention it.

Thank you in advance,

Evan

**Comment by EvanHLZ** [comment]
Hello,

I wonder where is the file named 'MNL_DA-00008.pt' appeared in the codes, it seems you didn't mention it.
Is it the pre-trained weights file 'weights_adapting_KonIQ10k.pt' or 'weights_adapting_SPAQ.pt'?

Thank you in advance,

Evan

**Comment by wzhsysu** [comment]
Yes, you are right

**Comment by nicolasch96** [comment]
Hello,

Sorry but I do not have access to the semantic scholar membership, is it possible to send it maybe by email?

Thank you,

Nicolas
